### DISCUSSION TITLE
Is Grapesjs stable? 3,000+ Eslint errors on first yarn build

### DISCUSSION THREAD

**Discussion by prodkt** [GrapesJS/grapesjs#5064] [Q&A]
I've looked over a couple bugs that had slightly started into eslint questions but they were ultimately unresolved and closed due to inactivity. I'm curious though as I've never encountered a package so popular so riddled with these errors on first yarn && yarn build. It on surface looks like a great option to start as a framework and foundation but this now feels like I'd be entering into and committing to managing more debt than it'd ever be worth.

Thats my largest concern at the moment. Is Grapesjs a viable solution to start a project with, and if so how do I explain these errors to anyone on my team before they themselves explore it? Without some messaging first here I don't think I'd be able to suggest it. Something like this would draw too many red flags in the first few minutes it would be shot down right there.

Thoughts? How have others overcome the initial roadblock of errors?

**Comment by artf** [comment]
Can you elaborate more on where/how do you see these eslint errors? 
By using grapesjs in a project? By developing grapesjs? Exact steps to generate those errors?
